cmd/statusd/utils.go

```diff
@@ -1414,6 +1418,29 @@ func testJailFunctionCall(t *testing.T) bool {
 	return true
 }
 
+func testNodeOffline(t *testing.T) bool {
+
+	StopNode()
+	//time to sync
+	time.Sleep(10 * time.Second)
+
+	loginResponse := common.APIResponse{}
```

Comment:
`var loginResponse common.APIResponse` is an equivalent and more reader friendly style. ;)

Reply:
Arguable. I prefer `:=` style, for example.